projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/ProjectorStatusWidget.kt

```diff
@@ -96,23 +91,45 @@ class ProjectorStatusWidget(project: Project)
     }
   }
 
-  private fun updateText() = "Projector"
+  private fun updateText() = "Projector" //  (${clientsCount()})"
```

Comment:
As we've decided not to implement connected clients count for now, I think it's better to revert such changes and add them in the next PR.

Also, fmpov, it's better not to output client count when Projector isn't running.

Reply:
Fixed